src/components/visualizations/table/TableVisualization.tsx

```diff
@@ -632,24 +632,29 @@ export class TableVisualizationClass
     }
 
     private enableTotalColumn(columnIndex: number, totalType: AFM.TotalType): void {
-        const updatedTotals = addMeasureIndex(this.props.totalsWithData, this.props.headers, totalType, columnIndex);
+        const updatedTotals =
+            addMeasureIndex(this.props.totalsWithData, this.props.headers, totalType, columnIndex);
 
         this.onTotalsEdit(updatedTotals);
     }
 
     private disableTotalColumn(columnIndex: number, totalType: AFM.TotalType): void {
-        const updatedTotals = removeMeasureIndex(this.props.totalsWithData, this.props.headers, totalType, columnIndex);
+        const updatedTotals =
+            removeMeasureIndex(this.props.totalsWithData, this.props.headers, totalType, columnIndex);
 
         this.onTotalsEdit(updatedTotals);
     }
 
-    private renderTooltipHeader(header: TableHeader, columnIndex: number, columnWidth: number)
+    private renderTooltipHeader(header: IMappingHeader, columnIndex: number, columnWidth: number)
         : (props: ITableVisualizationProps) => JSX.Element {
+        const { headers } = this.props;
+
         const headerClasses: string = getHeaderClassNames(header);
+        const headerName = getMappingHeaderName(header);
         const bubbleClass: string = uniqueId('table-header-');
         const cellClasses: string = classNames(headerClasses, bubbleClass);
-        const sort: ISortObj = this.getSortObj(header, columnIndex);
-        const columnAlign: Align = getColumnAlign(header);
+        const sort: ISortObj = this.getSortObj(headers[columnIndex], columnIndex);
```

Comment:
`.getSortObj(headers[columnIndex], columnIndex);` looks odd. especially using of columnIndex